pages/projects.tsx

```diff
@@ -1,17 +1,27 @@
 import next from 'next';
-import React, { useState } from 'react';
 import Navbar from '../components/layout/navbar/navbar';
 import style from '../styles/projects.module.scss';
+import React, { useEffect, useState } from 'react';
 import ProjectPage from '../components/projectPage/projectPage';
-import Footer from '../components/layout/footer/footer';
 import Layout from '../components/layout/layout';
+import { receivedProjects } from '../store/projectsSlice';
+import { getUsers } from '../services/user';
+import { useAppDispatch, useAppSelector } from '../store/hooks';
 
-export default function Vast() {
+export default function Projects(props) {
   const [currentHeight, setCurrentHeight] = useState(0);
-  console.log(currentHeight);
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    getUsers().then((products) => {
```

Comment:
can we add a `.catch` block and log the error? later we can add a field for managing the error (to the slice or in this component)